index.js

```diff
@@ -112,6 +112,12 @@
         // this is given meaning in componentDidMount
         __outsideClickHandler: function() {},
 
+        getDefaultProps: function() {
+          return {
+            excludeScrollbar: config && config.excludeScrollbar
```

Comment:
with this, https://github.com/Pomax/react-onclickoutside/blob/master/index.js#L171 will always exist, so we shouldn't need that `|| false` fallback anymore (with a comment `// pulls from defaultProps`